ADVANCED.md

```diff
@@ -386,13 +386,13 @@ type AnchorProps = React.AnchorHTMLAttributes<HTMLAnchorElement>
 type RouterLinkProps = Omit<NavLinkProps, 'href'>
 
 const Link = <T extends {}>(
-props: LinkProps & T extends RouterLinkProps ? RouterLinkProps : AnchorProps
+  props: LinkProps & T extends RouterLinkProps ? RouterLinkProps : AnchorProps
 ) => {
-if ((props as RouterLinkProps).to) {
-return <NavLink {...props as RouterLinkProps} />
-} else {
-return <a {...props as AnchorProps} />
-}
+  if ((props as RouterLinkProps).to) {
+    return <NavLink {...props as RouterLinkProps} />
+  } else {
+    return <a {...props as AnchorProps} />
+  }
 }
 
 <Link<RouterLinkProps> to="/">My link</Link> // ok
```

Comment:
All of these lines need a semicolon at the end. Otherwise this is invalid syntax

Reply:
You were correct, adding semicolons at the end and running `yarn format` helped.